src/Component/Symbolizer/ColorMapEditor/ColorMapEditor.tsx

```diff
@@ -94,6 +94,7 @@ export const ColorMapEditor: React.FC<ColorMapEditorProps> = ({
         type: 'ramp'
       };
     }
+    // @ts-ignore
```

Comment:
Is this problem fixed with #2132 maybe?

Reply:
Probably, I'll rebase against this once merged.